packages/web-components/src/components/va-featured-content/va-featured-content.tsx

```diff
@@ -1,4 +1,15 @@
 import { Component, Host, h, Prop, Element, State } from '@stencil/core';
+import { iterateAddAttrs, SlotTarget } from '../../utils/slot-child-class-append';
+
+const targetingArray:SlotTarget[] =[
+  { slotName: 'headline',
+    selectorClasses: [["h3","usa-summary-box__heading",true]],
+    selectorId: [['h3', 'summary-box-key-information']]
```

Comment:
This variable is storing a config for the utility.  I noticed in the example given to me, that some elements, were assigned id's as well as classes. Since I was told the VFS teams would not be responsible for assigning class attributes, I assumed they would also not assign corresponding ID's. This value can be configured  on a component slot by component slot basis. What you see here is what my interpretation of the needed IDs specific to the [featured-content example](https://designsystem.digital.gov/components/summary-box/#component-code). Also, since IDs are meant to be unique, it targets only the first selector that matches...as a safety. The selectors can be customized to fit the needs, and if the VFS/content teams give the id's to the slotted content themselves this step can/is omitted. Classes are different as they can be repeated, each matched selector ( in this case h3) would get assigned `usa-summary-box__heading`

This is trying to handle assignments from the premise that:
1) VFS/content team  will slot the elements providing no guidance other than the slot-name
2) They could compose (pass any element or grouping of element)
3) My interpretation of [summary-box guidance](https://designsystem.digital.gov/components/summary-box/#guidance), which leaves a lot open possibilities